src/pages/iou/IOUModal.js

```diff
@@ -83,7 +85,7 @@ const defaultProps = {
     myPersonalDetails: {
         localCurrencyCode: CONST.CURRENCY.USD,
     },
-    iouType: '',
+    iouType: CONST.IOU_TYPE.REQUEST,
```

Comment:
@nickmurray47 I have added the global constants for the IOU_TYPE and substituted it where I could. Something to be aware of though is that I changed the default to `request` as it would not be very clear if it would be empty string. 

Also for `split` we use the `hasMultipleParticipants` so it is not actual paymentType, but it will be better to have the constant there for future uses.

Reply:
That sounds good to me. I'm hoping to eventually refactor the use of `hasMultipleParticipants` too but I think this is fine for now.